contracts/storage/RibbonGammaVaultStorage.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+import {Vault} from "../libraries/Vault.sol";
+
+abstract contract RibbonGammaVaultStorageV1 {
+    /// @notice Stores the user's pending deposit for the round
+    mapping(address => Vault.DepositReceipt) public depositReceipts;
+
+    /// @notice On every round's close, the pricePerShare value of an rTHETA token is stored
+    /// This is used to determine the number of shares to be returned
+    /// to a user with their DepositReceipt.depositAmount
+    mapping(uint256 => uint256) public roundPricePerShare;
+
+    /// @notice Stores pending user withdrawals
+    mapping(address => Vault.Withdrawal) public withdrawals;
+
+    /// @notice Vault's parameters like cap, decimals
+    Vault.VaultParams public vaultParams;
+
+    /// @notice Vault's lifecycle state like round and locked amounts
+    Vault.VaultState public vaultState;
+
+    /// @notice Fee recipient for the performance and management fees
+    address public feeRecipient;
+
+    /// @notice role in charge of weekly vault operations such as rollToNextOption and burnRemainingOTokens
+    // no access to critical vault changes
+    address public keeper;
+
+    /// @notice Performance fee charged on premiums earned in rollToNextOption. Only charged when there is no loss.
+    uint256 public performanceFee;
+
+    /// @notice Management fee charged on entire AUM in rollToNextOption. Only charged when there is no loss.
+    uint256 public managementFee;
+
+    /// @notice Amount locked for scheduled withdrawals last week;
+    uint256 public lastQueuedWithdrawAmount;
```

Comment:
Are we propagating the changes from https://github.com/ribbon-finance/ribbon-v2/pull/238?